elastalert/alerts.py

```diff
@@ -220,5 +210,36 @@ def alert(self, matches):
             raise EAException("Error creating JIRA ticket: %s" % (e))
         logging.info("Opened Jira ticket: %s" % (self.issue))
 
+    def create_title(self, matches):
+        if 'alert_subject' in self.rule:
+            return self.create_custom_title(matches)
+
+        return self.create_default_title(matches)
+
+    def create_custom_title(self, matches):
+        alert_subject = self.rule['alert_subject']
+
+        if 'alert_subject_args' in self.rule:
+            alert_subject_args = self.rule['alert_subject_args']
+            alert_subject_values = [matches[0][arg] for arg in alert_subject_args]
+            return alert_subject.format(*alert_subject_values)
```

Comment:
I have added now default value for get. Not sure what else it would make sense at this point.
Ensuring that the `alert_subjects_args` contains the exact number of elements to satisfy all `{#}` fields in `alert_subject`?